<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>992</width>
    <height>699</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>992</width>
    <height>679</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>11198</width>
    <height>779</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">//background-color: #eb493d;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>60</width>
      <height>681</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>60</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">//background-color: #f55516;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="pushButtonAccount">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>36</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../resources/images/accountIcon-removebg.png</normaloff>../resources/images/accountIcon-removebg.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonHome">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>56</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../resources/images/homeIcon.png</normaloff>../resources/images/homeIcon.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonAnime">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>102</y>
       <width>40</width>
       <height>50</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../resources/images/rankingIcon-removebg.png</normaloff>../resources/images/rankingIcon-removebg.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>42</width>
       <height>42</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonManager">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>158</y>
       <width>40</width>
       <height>46</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../resources/images/manageIcon-removebg.png</normaloff>../resources/images/manageIcon-removebg.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>38</width>
       <height>38</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonQuit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>625</y>
       <width>40</width>
       <height>46</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../resources/images/quitIcon-removebg.png</normaloff>../resources/images/quitIcon-removebg.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>38</width>
       <height>38</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>908</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>50</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>0</y>
       <width>271</width>
       <height>50</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLineEdit" name="lineEditSearch">
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonSearch">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>SEARCH</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>110</y>
      <width>791</width>
      <height>511</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>80</y>
       <width>621</width>
       <height>341</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="page">
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>10</y>
         <width>201</width>
         <height>71</height>
        </rect>
       </property>
       <property name="text">
        <string>Trang 0 </string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2">
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>171</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Trang 1</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_3">
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>181</width>
         <height>71</height>
        </rect>
       </property>
       <property name="text">
        <string>Trang 2</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_4">
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>10</y>
         <width>161</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Trang 3</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
